utilities/make_agasc_supplement.py

```diff
@@ -0,0 +1,49 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+"""
+Create the initial version of agasc_supplement.h5.
+
+This file is a supplement to the stable AGASC to inform star selection
+and star catalog checking.
+
+This script simply creates the initial file that has only bad
+stars from two sources:
+ - starcheck bad star list
+   https://github.com/sot/starcheck/blob/master/starcheck/data/agasc.bad
+ - GAIA high proper motion file $SKA/analysis/gaia/agasc_gaia_xmatch_PM_gt_50mas.fits.gz
+   See: https://nbviewer.jupyter.org/url/cxc.cfa.harvard.edu/mta/ASPECT/ipynb/star_selection/gaia
+        GAIA guide star crossmatch.ipynb
+
+"""
+import os
+from pathlib import Path
+
+import numpy as np
+
+from astropy.table import Table
+
+HOME = Path(os.environ['HOME'])
+SKA = Path(os.environ['SKA'])
+
+agasc_ids = []
+sources = []
+
+# Starcheck bad star list is not installed anywhere so just grab from local git repo
```

Comment:
Well, it is installed in package data but..

Reply:
In ska3-flight I cannot `import starcheck`, so that file being in the package data doesn't help me too much.

Reply:
Right, in ska3-flight alone there is no starcheck.